### DISCUSSION TITLE
Find Step Definition Usages

### DISCUSSION THREAD

**Discussion by bgoggan** [reqnroll/Reqnroll#168] [Q&A]
Has anyone encountered the below error when using the above feature?

"Unable to find step definition usages: the project is not detected to be a Reqnroll project or it is not initialized yet."

I'm using the Reqnroll extension 2024.2.93 for VS 2022 and we're using Reqnroll v2.0.2

I know the step exists and is used in at least one feature file. Now, the step definition is kept in a separate project from the feature itself. I'm not sure if the "Find" feature has any limitations there. The test runs fine and the step is identified correctly on the feature.

**Comment by gasparnagy** [comment]
Seems like a bug. Could you please make a simple repro for this and report it as an issue?

**Reply by bgoggan** [reply]
@gasparnagy  Hey, sorry this has taken so long. I created a quick sample project and you should be able to access it from the link below. Let me know if you're not able too. Basically, I created a feature in the FindStepDefs project called StepDefsInDifferentProject and then in the 2nd project, StepDefs, I have step definition for that feature. I have the reqnroll.json setup with the assembly reference and when you're in the feature file you can dive into the step.....but, if you right click the step in the step definition file and select the Find Step Definition Usages an error is returned they can't be found. Let me know what you think. Thanks.

https://github.com/bgoggan/FindStepDefs
